sql/delete.go

```diff
@@ -18,11 +18,57 @@
 package sql
 
 import (
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/proto"
 	"github.com/cockroachdb/cockroach/sql/parser"
-	"github.com/cockroachdb/cockroach/util"
 )
 
-// Delete TODO(pmattis): document.
+// Delete deletes rows from a table.
 func (p *planner) Delete(n *parser.Delete) (planNode, error) {
-	return nil, util.Errorf("TODO(pmattis): unimplemented")
+	tableDesc, err := p.getAliasedTableDesc(n.Table)
+	if err != nil {
+		return nil, err
+	}
+
+	primaryKeyColumn := tableDesc.Columns[0]
+	primaryKeyPrefix := encodeIndexKeyPrefix(tableDesc.ID, primaryKeyColumn.ID)
```

Comment:
This isn't correct. The primary key is not necessarily a single column. The primary key index is `tableDesc.Indexes[0]`.